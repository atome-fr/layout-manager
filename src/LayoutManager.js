import React from 'react';

export default function LayoutManager(props) {
    return (
        <div className="containerView">
            {
                props.children
            }
        </div>
    );
}