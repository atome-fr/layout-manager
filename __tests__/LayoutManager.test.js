import React from 'react';
import LayoutManager from '../src/LayoutManager';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('LayoutManager Component', () => {

    describe('UI Render Test', () => {

        it('should render with all the children component', () => {
            const wrapper = shallow(<LayoutManager>
                <div>View 1</div>
            </LayoutManager>);
            const tree = toJson(wrapper);

            expect(tree).toMatchSnapshot();
            expect(wrapper.find("div").length).toBe(2);
        });
    });

});