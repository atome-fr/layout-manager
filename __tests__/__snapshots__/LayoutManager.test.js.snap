// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutManager Component UI Render Test should render with all the children component 1`] = `
<div
  className="containerView"
>
  <div>
    View 1
  </div>
</div>
`;
